lib/trento/infrastructure/commanded/event_handlers/activity_log_event_handler.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Trento.Infrastructure.Commanded.EventHandlers.ActivityLogEventHandler do
+  @moduledoc """
+  Event handler responsible to log activity from emitted domain events
+  """
+
+  use Commanded.Event.Handler,
+    application: Trento.Commanded,
+    name: "activity_log_event_handler"
+
+  alias Trento.ActivityLog.ActivityLogger
+
+  def handle(event, metadata) do
+    ActivityLogger.log_activity(%{
+      event: event,
+      metadata: metadata
+    })
+
+    :ok
```

Comment:
A doubt I have here is: if for any reason logging a specific interesting domain event fails, we'd be continuing silently.

If we want to be more reliable and have the event handler retry, we might want to not always return `:ok`.
That means having `ActivityLogger.log_activity` possibly return an :ok/:error tuple.

I guess we can have this improvement as a followup.